fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/Action.java

```diff
@@ -12,7 +12,6 @@ public enum Action {
   SETUP("setup"),
   UPDATE("update_properties"),
   STATE("state"),
-  GREET("greet"),
```

Comment:
Why are you deleting this, I think this is used for election but @ahominal will know more than me

Reply:
That was not intended (done by mistake). I've put it back.